integration_tests/test_mesh_lens_correction.py

```diff
@@ -60,15 +171,24 @@ def test_coarse_mesh_failure(render, tmpdir_factory):
     example_for_input['output_dir'] = outdir
     example_for_input['out_html_dir'] = out_html_dir
     example_for_input['z_index'] = 101
+    example_for_input['input_stack'] = raw_lens_stack_1
+    # just testing code, let's keep it fast
+    example_for_input['downsample_scale'] = 0.15
+    example_for_input['match_collection'] = raw_lens_matches_1b
+    # this will cover delete_matches_if_exist()
+    example_for_input['rerun_pointmatch'] = True
     outjson = os.path.join(outdir, 'mesh_lens_out0.json')
 
     meshmod = MeshLensCorrection(
             input_data=example_for_input,
             args=['--output_json', outjson])
 
+    # will raise Exception because there aren't many matches
     with pytest.raises(MeshLensCorrectionException):
         meshmod.run()
 
+    rmtree(outdir)
```

Comment:
Not sure why this is necessary when outdir is created by a `tempdir_factory` fixture

Reply:
in my local test environment, these directories seem to persist

Reply:
if we're going to abandon python 2.x we could start using context manager:
```
with TemporaryDirectory() as output_dir:
   blah
```

Reply:
Right, I see that now.  Would it make more sense to do a function-scoped fixture in this case, then?

Reply:
sure, i'll do that. it will be cleaner